backend/src/main/kotlin/blue/mild/covid/vaxx/routes/PatientRoutes.kt

```diff
@@ -63,21 +63,26 @@ fun NormalOpenAPIRoute.patientRoutes() {
             captchaService.verify(recaptchaToken, request.determineRealIp())
             logger.debug { "Captcha token verified. Validating isin." }
 
-            val isinValidationResult = isinValidationService.validatePatientIsin(patientRegistration)
-
-            val patientIsinId: String? = when (isinValidationResult.status) {
-                IsinValidationResultStatus.PATIENT_FOUND ->
-                    isinValidationResult.patientId
-                IsinValidationResultStatus.PATIENT_NOT_FOUND ->
-                    throw IsinValidationException(isinValidationResult)
-                IsinValidationResultStatus.WAS_NOT_VERIFIED -> {
-                    logger.warn { "Patient was not validated in isin due to some problem. Skipping isin validation." }
-                    null
+            val patientIsinId: String? = if (patientRegistration.personalNumber != null) {
```

Comment:
Takze rikame, ze kdyz ma zaskrtnuto Jsem cizinec, tak nikdy nema vyplnene rodne cislo, jo? Jakoze fakticky on treba zapise sve rodne cislo do cisla pojistence, ale my to proste evidujeme jako cislo pojistence, nikoliv rodne cislo. Je to tak?

Reply:
Ano, češi vyplňují personal number, cizinci insurance number, které může být v jakémkoli formátu